tests/tracking/fluent/test_fluent.py

```diff
@@ -700,48 +733,30 @@ def test_search_runs_attributes(search_runs_output_format):
     reason="Skinny client does not support the np or pandas dependencies",
 )
 def test_search_runs_data():
-    import numpy as np
-    import pandas as pd
-
-    runs = [
-        create_run(
-            metrics=[Metric("mse", 0.2, 0, 0)],
-            params=[Param("param", "value")],
-            tags=[RunTag("tag", "value")],
-            start=1564675200000,
-            end=1564683035000,
-        ),
-        create_run(
-            metrics=[Metric("mse", 0.6, 0, 0), Metric("loss", 1.2, 0, 5)],
-            params=[Param("param2", "val"), Param("k", "v")],
-            tags=[RunTag("tag2", "v2")],
-            start=1564765200000,
-            end=1564783200000,
-        ),
-    ]
+    runs, data = create_test_runs_and_expected_data("pandas")
     with mock.patch("mlflow.tracking.fluent._paginate", return_value=runs):
         pdf = search_runs()
-        data = {
-            "status": [RunStatus.FINISHED] * 2,
-            "artifact_uri": [None] * 2,
-            "run_id": [""] * 2,
-            "experiment_id": [""] * 2,
-            "metrics.mse": [0.2, 0.6],
-            "metrics.loss": [np.nan, 1.2],
-            "params.param": ["value", None],
-            "params.param2": [None, "val"],
-            "params.k": [None, "v"],
-            "tags.tag": ["value", None],
-            "tags.tag2": [None, "v2"],
-            "start_time": [
-                pd.to_datetime(1564675200000, unit="ms", utc=True),
-                pd.to_datetime(1564765200000, unit="ms", utc=True),
-            ],
-            "end_time": [
-                pd.to_datetime(1564683035000, unit="ms", utc=True),
-                pd.to_datetime(1564783200000, unit="ms", utc=True),
-            ],
-        }
+        # data = {
```

Comment:
feel free to delete this. It's covered in your refactoring.

Reply:
ahh, missed that, thanks